Simplenote/AccountRemote.swift

```diff
@@ -8,32 +8,26 @@ class AccountRemote: Remote {
     /// Send verification request for specified email address
     ///
     func verify(email: String, completion: @escaping (_ result: Result<Data?, RemoteError>) -> Void) {
-        guard let request = verificationURLRequest(with: email) else {
-            completion(.failure(RemoteError.urlRequestError))
-            return
-        }
+        let request = verificationURLRequest(with: email)!
 
         performDataTask(with: request, completion: completion)
     }
 
     /// Send account deletion request for user
     ///
     func requestDelete(_ user: SPUser, completion: @escaping (_ result: Result<Data?, RemoteError>) -> Void) {
-        guard let request = deleteRequest(with: user) else {
-            completion(.failure(RemoteError.urlRequestError))
-            return
-        }
+        let request = deleteRequest(with: user)!
 
         performDataTask(with: request, completion: completion)
     }
 
     // MARK: URL Requests
 
     private func verificationURLRequest(with email: String) -> URLRequest? {
-        guard let base64EncodedEmail = email.data(using: .utf8)?.base64EncodedString(),
-              let verificationURL = URL(string: SimplenoteConstants.verificationURL) else {
+        guard let base64EncodedEmail = email.data(using: .utf8)?.base64EncodedString() else {
             return nil
         }
+        let verificationURL = URL(string: SimplenoteConstants.verificationURL)!
```

Comment:
If we force-unwrap the whole `URLRequest` later, we are better to force unwrap `base64EncodedEmail` here and change method signature to return non-optional request.